### DISCUSSION TITLE
The dreaded "sane defaults" topic

### DISCUSSION THREAD

**Discussion by AzumaHazuki** [lxqt/lxqt#2120] [Ideas]
So...I'm the writer of the Clearlooks (and Leech) themes, and was very pleasantly surprised to see them accepted into LXQt officially :) Huge thanks to Tsujan for all your help in getting them off the ground!

I know someone is probably going to say something along the lines of "what you're proposing is the distro's job," _but_ as a counterpoint, I almost never see a distro doing this job, and usually they do it poorly (Xubuntu being the worst offender in my book...).

Basically, I think LXQt's main problem with average end users is the out of the box experience is a bit...disjointed. Part of the goal of writing the Clearlooks theme was specifically to help LXQt integrate into the desktop Linux world better, since Clearlooks is more or less the Win95/98/2K style of Linux. 

Do you all remember how Xubuntu and Gnome-2.x Ubuntu _used_ to look? Two panels, with the Show Desktop button, taskbar, and maybe a pager on the bottom one and the menu, launchers, a big separator, systray, and clock along the top panel? I think this is the "sane default" we should be going for in terms of layout. It's instantly recognizable for any veteran, it's easy to use for a newbie, and it's a nice balance of screen real estate and available actions.

I was also wondering if we could add Clearlooks-Phenix GTK2/3 themes and a non-KDE-dependent version of QtCurve as dependencies and ship, for example, the Cleanlooks QtCurve style as a default, though Fusion works well enough too (or perhaps we could integrate KVantum and use Tsujan's KVantum LXQt style as default instead of Frost?). 

I've attached my daily-driver setup along these lines as an image to give an illustration of what I'm talking about. Panels are 24px. high with 16px icons, task manager entries are 200px instead of the default 400, and the logout button up there is actually a second Quicklaunch.

![LXQt](https://user-images.githubusercontent.com/67122280/141032070-ee2c67ea-c6f5-4cc5-8de1-fddddc47fcb7.jpg)

**Comment by tsujan** [comment]
> I was also wondering if we could add Clearlooks-Phenix GTK2/3 themes and a non-KDE-dependent version of QtCurve as dependencies...

We should never introduce false dependencies upstream. These are distro jobs (= downstream).

>  I know someone is probably going to say something along the lines of "what you're proposing is the distro's job,"...

Yes.

Please also note that what's ugly to you may be good to someone else and conversely. LXQt is a very flexible DE with lots of configurations. Users who don't want to touch config dialogs usually choose Gnome and nothing else.

**Comment by AzumaHazuki** [comment]
Okay but...in the same way choosing not to act is itself an action, the defaults of any DE are in effect its public face, its first impression. LXQt does not make a good first impression, even compared to what you get when you install vanilla Xfce or Gnome.

And this is about more than just appearances; the old-school *buntu setup lasted so long because _it worked,_ in several senses of the word. Its Fitts' Law compliant (mostly), it's intuitive, it's spare without being barren, and aside from the complete lack of anything in the middle of the top panel, it makes excellent use of space. Much functionality is exposed to the user immediately, in discoverable and low-effort ways. "Start stuff" in the upper left, "do stuff" at the bottom, "tweak stuff" and "okay, i'm done doing stuff" in the upper right.

The default theme is less important than the default layout, I think...though again, Frost as the default is an odd choice, something like Android 12 going back to Holo and pumping the default DPI/scaling up way too high.

**Reply by tsujan** [reply]
I'll have no objection to any decision that may be made about changing LXQt defaults, whether they're about themes or layouts (but not in the codes). A good discussion can be started here. I'm only against upstream false dependencies.

**Reply by stefonarch** [reply]
I agree with most. In cases like Lubuntu most is ok, but when somebody just installs lxqt-core package in most distros they will get not the best first  experience. One I remember was menu search not closed after action  (if I remember  well this is fixed in the code), another is an annoying message about shortcut conflicts with xfwm4, and I've seen many user who have no idea what a window manager is for.

So we could agree on a default layout in /etc/skel/ or similar? 

I recently tried out emmabuntus, a distro which ships lxqt and xfce as default, and uses a welcome window with settings to configure some things at first login. We already have that for lxqt-powermanager.

**Reply by tsujan** [reply]
> So we could agree on a default layout in /etc/skel/ or similar?

Why /etc/skel/? For example, we can change the default layout of Panel with our default `panel.conf` — unless I'm missing something.

**Reply by stefonarch** [reply]
I was missing something...
